python/fastsim/__init__.py

```diff
@@ -9,7 +9,7 @@
 from . import fastsimrust
 from . import fastsimrust as fsr
 from . import parameters as params
-from . import utilities as utils
+from .utils import utilities as utils
```

Comment:
@robinsteuteville 
Try `from . import utils` here...

Reply:
... and in utilities/__init__.py add `from .utilities import *` and check out the effect :)

Reply:
It's subtle, but this doesn't change anything about the package's namespace, and allows other utilities in the directory to be exposed as well (like if we wanted to expose the other file's functions under `fastsim.utils`)

Reply:
updated!